EIPS/eip-7823.md

```diff
@@ -57,44 +57,44 @@ Since MODEXP was introduced in the Byzantium hard fork, an analysis has been con
 
 | `input_of_BASE` | count |
```

Comment:
Are these bits or bytes? `input_of_BASE`

Reply:
Good catch. Thank you. https://github.com/ethereum/EIPs/pull/9284